docs/fsharp/language-reference/symbol-and-operator-reference/arithmetic-operators.md

```diff
@@ -20,6 +24,30 @@ The following table summarizes the binary arithmetic operators that are availabl
 |`%` (remainder, rem)|Returns the remainder of a division operation. The sign of the result is the same as the sign of the first operand.|
 |`**` (exponentiation, to the power of)|Possible overflow condition when the result exceeds the maximum absolute value for the type.<br /><br />The exponentiation operator works only with floating-point types.|
 
+The unchecked behavior does not throw exceptions when overflow occurs, making it less safe for arithmetic on large or edge-case values.
+
+### Checked Arithmetic Operators
+
+The following table summarizes the binary arithmetic operators that are available for **Checked Arithmetic** with unboxed integral types. Checked operators ensure that calculations are verified for overflow or underflow, providing safer arithmetic for critical applications.
+
+| Binary Operator | Notes                                                                                  |
+|------------------|----------------------------------------------------------------------------------------|
+| `+` (addition, plus) | Throws an <xref:System.OverflowException> if the result exceeds the maximum absolute value supported by the type. |
```

Comment:
Since negative numbers exist, I think both overflow and underflow is possible for all 3 of the operators.